packages/react/src/components/Table/TableBody/TableBodyRow/TableBodyRow.jsx

```diff
@@ -236,6 +245,27 @@ const TableBodyRow = ({
           />
         </span>
       </TableCell>
+    ) : hasRowSelection === 'single' && showRadioButton ? (
+      <TableCell
+        className={`${prefix}--radiobutton-table-cell`}
+        key={`${id}-row-selection-cell`}
+        onChange={isSelectable !== false ? () => onRowSelected(id, !isSelected) : null}
+        onClick={(e) => e.stopPropagation()}
```

Comment:
In the story, the cell around the radio button does not trigger a row click or a row selection. Is this stopPropagation causing that?

Reply:
> In the story, the cell around the radio button does not trigger a row click or a row selection. Is this stopPropagation causing that?

This is the same design as the checkbox in `multi` select. It looks like we can select either the table row or the checkbox/radio button.  Do we want to update it?